middle_end/flambda2/from_lambda/lambda_to_flambda_primitives_helpers.ml

```diff
@@ -24,13 +24,12 @@ module VB = Bound_var
 
    let bounds_check ~width ~string_length_in_bytes ~index_in_bytes :
    bounds_check_result = let index_in_bytes = Targetint_31_63.to_targetint
-   index_in_bytes in if Targetint_31_63.Imm.compare index_in_bytes
-   Targetint_31_63.Imm.zero < 0 then Out_of_range else let result_size_in_bytes
-   = Targetint_31_63.Imm.of_int
-   (Flambda_primitive.byte_width_of_string_accessor_width width) in (* We are
-   careful here to avoid overflow for ease of reasoning. *) let
-   highest_index_allowed = Targetint_31_63.Imm.sub string_length_in_bytes
-   result_size_in_bytes in if Targetint_31_63.Imm.compare index_in_bytes
+   index_in_bytes in if Targetint_31_63.compare index_in_bytes
+   Targetint_31_63.zero < 0 then Out_of_range else let result_size_in_bytes =
+   Targetint_31_63.of_int (Flambda_primitive.byte_width_of_string_accessor_width
+   width) in (* We are careful here to avoid overflow for ease of reasoning. *)
+   let highest_index_allowed = Targetint_31_63.sub string_length_in_bytes
+   result_size_in_bytes in if Targetint_31_63.compare index_in_bytes
```

Comment:
Do we really want to keep that comment ? If so, could we try and format it so that it is at least readable ?

Reply:
I'll remove this and another piece of old code in a separate PR.